.editorconfig

```diff
@@ -1,8 +1,8 @@
 [*]
-charset = utf-8-bom
-end_of_line = crlf
+charset = utf-8
+end_of_line = lf
 trim_trailing_whitespace = false
-insert_final_newline = false
+insert_final_newline = true
```

Comment:
This triggered a bunch of new warnings, mind resolving them? I agree with the change, we should apply them to our remaining dotnet repos later

Reply:
Added a new commit fixing a lot of these formating issues.

Used this feature from Jetbrains (Ctrl+Alt+Shift+L)

![image](https://user-images.githubusercontent.com/114933129/209186058-6a6db85d-ad51-4215-af0a-b2301884a48f.png)

There were a lot of other changes that I reverted from that cleanup because I want to discuss them with you later

Reply:
Sounds good! Resolving